code/game/objects/items/weapons/storage/storage.dm

```diff
@@ -417,6 +417,15 @@
 		return
 	..()
 
+	if(istype(W,/obj/item/stack/package_wrap))
```

Comment:
I feel like this should be in the `package_wrap`'s `preattack` rather than here.

Reply:
I really don't understand the flow of the attack procs, putting it in the `package_wrap`'s `preattack` causes more issues than I'm willing to solve.
![image](https://user-images.githubusercontent.com/8468269/85218077-11e6e680-b397-11ea-9884-9ce0a9562206.png)

Reply:
You can `return TRUE` from `preattack` to prevent `attackby` from running.

Reply:
why you gotta be like this